scripts/checkmk/script.py

```
@@ -19,11 +19,12 @@
 import json
 import os
 
+
 def check_new_devices():
     # Get API path from environment variable, fallback to /tmp/api
     api_path = os.environ.get('NETALERTX_API', '/tmp/api')
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/api"

(S108)